libs/sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -2286,7 +2294,8 @@ private static void printStatement(String sql) {
 
             println(SqlFormatter.formatSql(statement));
             println("");
-            assertFormattedSql(statement);
+            Function<Node, String> formatter = random().nextBoolean() ? SqlFormatter::formatSql : SqlFormatter::formatSqlInline;
+            assertFormattedSql(statement, formatter);
```

Comment:
Could we just always assert both? Would avoid the randomization and reintroduction of junit4 for this module.